src/Akka.Hosting/AkkaHostedService.cs

```diff
@@ -17,11 +17,11 @@ internal sealed class AkkaHostedService : IHostedService
         private CoordinatedShutdown? _coordinatedShutdown; // grab a reference to CoordinatedShutdown early
         private readonly IServiceProvider _serviceProvider;
         private readonly AkkaConfigurationBuilder _configurationBuilder;
-        private readonly IHostApplicationLifetime _hostApplicationLifetime;
+        private readonly IHostApplicationLifetime? _hostApplicationLifetime;
         private readonly ILogger<AkkaHostedService> _logger;
 
         public AkkaHostedService(AkkaConfigurationBuilder configurationBuilder, IServiceProvider serviceProvider,
-            ILogger<AkkaHostedService> logger, IHostApplicationLifetime applicationLifetime)
+            ILogger<AkkaHostedService> logger, IHostApplicationLifetime? applicationLifetime)
```

Comment:
MAUI does not support `IHostApplicationLife`, we need to pass in `null` and disable termination hook.